docs/src/modules/components/AppFrame.js

```diff
@@ -122,15 +122,13 @@ const StyledAppBar = styled(AppBar, {
     }),
     ...(!disablePermanent && {
       [theme.breakpoints.up('lg')]: {
-        width: 'calc(100% - 240px)',
+        width: 'calc(100% - 250px)',
       },
     }),
     boxShadow: 'none',
-    borderStyle: 'solid',
+    borderBottom: '1px solid',
     borderColor:
       theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[100],
-    borderWidth: 0,
-    borderBottomWidth: 'thin',
```

Comment:
@hbjORbj I am not sure why this file change, do you know?

Reply:
@danilo-leal made this change. I think he just simplified the same style here?

Reply:
Okay, thanks.

Reply:
Both files, AppFrame and AppNavDrawer, to also address both comments (this and the one below), were tweaked because I was trying to have the Algolia search above it all once opened. The previous behavior was the AppNavDrawer being on the same plane as the search.

To address it, Benny suggested adding a `zIndex: 1000` to the NavDrawer, which worked to resolve the Algolia being on top of it all but then the AppFrame got above the NavDrawer. Thus, I had to tweak both in order to revert them to the previous look while continue to have Algolia above it all.

Reply:
But anyway! It seems you guys have already solved it all way better than I could have 😬 So, resolving this thread as well.